### ISSUE TITLE
[$10][MSFT-188] [Challenge Feed][Chrome] challenge page Image element has no alt attribute

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2856]
**Steps to Reproduce**
1) Got to the page "https://www.topcoder.com/challenges/<code>" 
2) Use to check accessibility compliance website 'http://www.cynthiasays.com/' and enter menederoty things with webpage URL e.g. Email ID, Web Page URL(https://www.topcoder.com/challenges/<code>),
Compliance mode(WCAG 2.0A)
3) Tick the term and condition 
4) Submit the Test you site button and observe the scan output.

**Expected Result**
Expect to alternative tag in all the image element in challenge page.

**Actual Result**
Challenge page image element has not alternation  attribute.
http://www.cynthiasays.com/ShowInstances.aspx?resultId=11429818

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.5 Images of Text
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: Achecker

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=8ab28edc%2Dd11f%2D4bb3%2Db9cf%2Da743b86a634f&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fkmurti%2FAccessibility%2F5

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096031 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096031 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096031 has been updated - it has been assigned to cunhavictor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096870 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096870 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu prize for this ticket is yet to be doubled.

**Comment by r0hit-gupta** [comment]
@veshu currently this issue affects only the page https://www.topcoder.com/challenges/30095347 because of the image in the description. Please review.

**Comment by crazyk07** [comment]
OOO

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
